source/expander.txt

```diff
@@ -0,0 +1,27 @@
+Expander
+========
+
+Expanders allow to dynamically hide or show information within a window or
+dialog. An expander can take a single widget that will be displayed when
+expanded.
+
+Expanders remain expanded until clicked again. When the state of an expander
+is changed, the "activate" signal is emitted.
+
+An expander can be programmatically expanded or retracted by passing `True` or
```

Comment:
What do you mean by "retraced"?

Reply:
I meant un-expanded, maybe "collapsed" is a better term?